packages/storage-plus/src/map.rs

```diff
@@ -170,9 +185,41 @@ where
         'a: 'c,
     {
         let mapped =
-            namespaced_prefix_range(store, self.namespace, min, max, order).map(deserialize_kv);
+            namespaced_prefix_range(store, self.namespace, min, max, order).map(deserialize_v);
```

Comment:
Let's make this `prefix_range_de`. Output should be `(K::Output, T)` as below

Reply:
Just created the corresponding fn, for completeness.